tests/ui_tools/test_boxes.py

```diff
@@ -743,57 +743,72 @@ def test_generic_autocomplete_user_mentions(
         )
```

Comment:
In the commit introducing stream_categories, note that while details can be useful, in the commit text it's more useful to write what you can't easily include in the commit (code) itself, such as why the change is useful/important.

Some of that why might be noting how it might be extended, but until the extra parameters are added, it's not important to refer to them by name - otherwise you couple this commit text to how it *will* be implemented, and either this commit text needs updating later (though it might be merged first!), or else the later code 'must' be named appropriately later (which one cannot always guarantee). Basically, it can be useful to describe what else it might be applied to, or perhaps an example, but otherwise focus the commit text on *this* commit.

I may have given an impression that something wasn't in the commit text which led to this; I believe I was referring to how the commit text didn't describe what this commit was doing (re switching to stream_categories).